src/commands/command-info.json

```diff
@@ -20,6 +20,9 @@
         "acl_categories": [
             "CONNECTION"
         ],
+        "command_tips": [
+            "NONDETERMINISTIC_OUTPUT"
+        ],
```

Comment:
i thought it would share the same logic with COMMAND. COMMAND is `NONDETERMINISTIC_OUTPUT`.
yes, due to the dict, maybe they should all be `NONDETERMINISTIC_OUTPUT_ORDER`...

Reply:
yes, i suppose the fact COMMAND used `NONDETERMINISTIC_OUTPUT` was a mistake.
i now realize why COMMAND INFO didn't have it.
it's because until recently COMMAND INFO didn't support 0 arguments, so the order was deterministic.
now you can just call COMMAND INFO with no arguments, which makes it behave the same as COMMAND without args.
so they should both be `NONDETERMINISTIC_OUTPUT_ORDER`